src/win32.mak

```diff
@@ -524,7 +524,7 @@ $G/dt.obj : $C\dt.h $C\dt.c
 	$(CC) -c -o$@ $(MFLAGS) $C\dt
 
 $G/dvec.obj : $C\dvec.d
-	$(HOST_DC) -c -of$@ $(DFLAGS) -betterC $C\dvec
+	$(HOST_DC) -c -of$@ $(DFLAGS) -betterC -mv=dmd.backend=$C $C\dvec
```

Comment:
Is this necessary when the file has the correct module declaration, `dmd.backend.dvec`?

Reply:
yes, because otherwise the compiler cannot find it.

Reply:
dvec.d is also used by dmc++ which has a different directory structure. The whole point of `-mv` is so different directory structures can be accommodated easily. We've discussed this innumerable times, let's just agree to disagree on the value of `-mv`.